docs/features/collection.adoc

```diff
@@ -69,7 +69,7 @@ logstash 7.10.1|✓|
 |Loglevel||✓|✓
 |https://github.com/openshift/enhancements/blob/196445c9d19b2159c9e8639e4428fa5a4c1b3577/enhancements/cluster-logging/forwarding-json-structured-logs.md[JSON Parsing]||✓|✓
 |Structured Index for Elasticsearch JSON parsing||✓|✓
-|Multiline error detection||✓|✓
+|https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/logforwarding/multiline-error-detection.md[Multiline error detection]|See feature document for languages supported by each collector|✓|✓
```

Comment:
The url here is not correct.

it should be `docs/features/logforwarding/multiline-error-detection.adoc` not `.md`

![not found](https://user-images.githubusercontent.com/3284044/229561475-f3896a2c-104c-4977-917f-06f68f184393.png)

Reply:
Thanks, nice catch.   done.